sdk/python/feast/base_feature_view.py

```diff
@@ -49,6 +49,8 @@ class BaseFeatureView(ABC):
     created_timestamp: Optional[datetime]
     last_updated_timestamp: Optional[datetime]
 
+    _entity_key_serialization_version: Optional[int]
```

Comment:
I'm assuming there would just be 2 versions, 1 b4 and 1 after. Maybe we can add a comment that details what values this can be?